packages/node/src/fork-dev-server.ts

```diff
@@ -20,7 +20,7 @@ export function forkDevServer(options: {
    */
   devServerPath?: string;
 }) {
-  let nodeOptions = process.env.NODE_OPTIONS;
+  let nodeOptions = process.env.NODE_OPTIONS ?? '--no-warnings';
```

Comment:
If this is not added, the terminal that runs `vc dev` shows the following message:

```
node:54367) [UNDICI-WS] Warning: WebSockets are experimental, expect them to change at any time.
(Use `node --trace-warnings ...` to show where the warning was created)
```

Reply:
This will only add `--no-warnings` if `NODE_OPTIONS` is not defined. It might be better to append `--no-warnings`:

```suggestion
  let nodeOptions = process.env.NODE_OPTIONS || '';
  
  if (!nodeOptions.includes('--no-warnings')) {
    nodeOptions  += ' --no-warnings';
  }
```

Secondly, `nodeOptions` is overwritten by lines 33, 35, and 41. Those should be appended too.

Thirdly, I'd like to land the `vc dev` ESM PR before making more changes to the dev server related bits. That PR touches `nodeOptions`: https://github.com/vercel/vercel/pull/9692/files#diff-5f9eca40d7958b4ff2cd6b5ebcd853b745fe4c76681941846d05a87d1f686c92L32.